### ISSUE TITLE
UnsatisfiedLinkError when matching State.CHECKING_RESUME_DATA enum in switch block

### ISSUE THREAD

**Issue by proninyaroslav** [aldenml/libtorrent4j#144]
**Version**: 2.0.1-7
**OS**: Android

Usage example: https://github.com/proninyaroslav/libretorrent/blob/fdfbd604e66c988c6d9aeef771b9a4eabaa3db22/app/src/main/java/org/proninyaroslav/libretorrent/core/model/session/TorrentDownloadImpl.java#L1344

Stack trace:
```
Rejecting re-init on previously-failed class java.lang.Class<org.proninyaroslav.libretorrent.core.model.session.TorrentDownloadImpl$1>: java.lang.UnsatisfiedLinkError: No implementation found for int org.libtorrent4j.swig.libtorrent_jni.torrent_status_checking_resume_data_get() (tried Java_org_libtorrent4j_swig_libtorrent_1jni_torrent_1status_1checking_1resume_1data_1get and Java_org_libtorrent4j_swig_libtorrent_1jni_torrent_1status_1checking_1resume_1data_1get__)
        at int org.libtorrent4j.swig.libtorrent_jni.torrent_status_checking_resume_data_get() (libtorrent_jni.java:-2)
        at void org.libtorrent4j.swig.torrent_status$state_t.<clinit>() (torrent_status.java:594)
        at void org.libtorrent4j.TorrentStatus$State.<clinit>() (TorrentStatus.java:711)
        at org.libtorrent4j.TorrentStatus$State[] org.libtorrent4j.TorrentStatus$State.values() (TorrentStatus.java:705)
        at void org.proninyaroslav.libretorrent.core.model.session.TorrentDownloadImpl$1.<clinit>() (TorrentDownloadImpl.java:1346)
        at void org.proninyaroslav.libretorrent.core.model.session.TorrentDownloadImpl$InnerListener.alert(org.libtorrent4j.alerts.Alert) (TorrentDownloadImpl.java:210)
        at void org.libtorrent4j.SessionManager.fireAlert(org.libtorrent4j.alerts.Alert, int) (SessionManager.java:1042)
        at void org.libtorrent4j.SessionManager.access$1200(org.libtorrent4j.SessionManager, org.libtorrent4j.alerts.Alert, int) (SessionManager.java:62)
        at void org.libtorrent4j.SessionManager$5.run() (SessionManager.java:1201)
        at void java.lang.Thread.run() (Thread.java:923)
```

**Comment by aldenml** [comment]
ACK, will take a look

**Comment by aldenml** [comment]
It seems that I was assembling the wrong binaries due to the recent changes in travis and OSS projects

**Comment by aldenml** [comment]
Hi @proninyaroslav, can you give `2.0.1-8` a try?

**Comment by proninyaroslav** [comment]
I confirm that it has been fixed in version 2.0.1-8.
